tests/lpm_test_interface.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/sh
```

Comment:
> This script is pure POSIX sh so it do not need to be as bash script.

This is the wrong solution to marking a script as POSIX sh. Please use /bin/sh exclusively for this, as that is the portable filename for the "sh" command.

Reply:
Please notice that currently NONE ot the OS distros (Linux/Solaris/*BSD) uses /usr separation and all those OSess distros uses /usr/bin/sh as standard POSIX sh location (Solaris and *BSD uses that more than decade).
Just look on Fedora or Debian packaging standards.
Fedora rpm currently **automatically** changed /bin/ to /usr/bin/ in all shells shebangs https://src.fedoraproject.org/rpms/redhat-rpm-config/blob/rawhide/f/brp-mangle-shebangs

Reply:
> Please notice that currently NONE ot the OS distros (Linux/Solaris/*BSD) uses /usr separation and all those OSess distros uses /usr/bin/sh as standard POSIX sh location (Solaris and *BSD uses that more than decade).

Clearly you have no idea what you're talking about in the slightest.

```
$ uname -a
FreeBSD build 14.0-RELEASE-p5 FreeBSD 14.0-RELEASE-p5 #0: Tue Feb 13 23:37:36 UTC 2024     root@amd64-builder.daemonology.net:/usr/obj/usr/src/amd64.amd64/sys/GENERIC amd64

$ file /usr/bin/sh /bin/sh
/usr/bin/sh: cannot open `/usr/bin/sh' (No such file or directory)
/bin/sh:     ELF 64-bit LSB pie executable, x86-64, version 1 (FreeBSD), dynamically linked, interpreter /libexec/ld-elf.so.1, for FreeBSD 14.0 (1400097), FreeBSD-style, stripped
```

Not only is /usr/bin/sh not the "standard POSIX sh location", it isn't a valid location at all. It doesn't exist. It never did and never has and most likely never will.

It's generally unlikely to find /usr/bin/sh except as a side effect of Linux-specific systems that happen to have implemented /bin as a symlink to /usr/bin, which isn't even universal on Linux either. You won't get very far on Alpine Linux, for example, if you assume that /usr/bin/sh exists.

This is why it is NOT the "standard POSIX sh location".

Reply:
Hi, guys
May I know if we are on the same page now, for the discussion above?